travis/test_server.py

```diff
@@ -133,6 +145,17 @@ def get_addons_path(travis_home, travis_build_dir, server_path):
     return addons_path
 
 
+def get_default_travis_addons_path():
+    '''
+
+    '''
+    return get_addons_path(
+        os.environ['HOME'],
+        os.environ['TRAVIS_BUILD_DIR'],
+        get_default_travis_server_path(),
+    )
+
+
```

Comment:
In process of change this method to `getaddons.py`
I saw that the original method wrapped exists in [test_server.py#L107-L133](https://github.com/OCA/maintainer-quality-tools/blob/a688099829eea9b520fd6e60c21f16c7e8433035/travis/test_server.py#L107-L133)